aws-xray/src/main/java/io/opentelemetry/contrib/awsxray/RemoteAttributesSpanProcessor.java

```diff
@@ -0,0 +1,58 @@
+/*
+ * Copyright The OpenTelemetry Authors
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+package io.opentelemetry.contrib.awsxray;
+
+import io.opentelemetry.api.common.AttributeKey;
+import io.opentelemetry.api.trace.Span;
+import io.opentelemetry.context.Context;
+import io.opentelemetry.sdk.trace.ReadWriteSpan;
+import io.opentelemetry.sdk.trace.ReadableSpan;
+import io.opentelemetry.sdk.trace.SpanProcessor;
+
+/**
+ * RemoteAttributesSpanProcessor copies attributes aws.remote.application and aws.remote.operation
+ * from the parent span to children after receiving them. These two attributes will be used as
+ * RemoteApplication and RemoteOperation dimension values in dependency metrics generation.
+ */
+public class RemoteAttributesSpanProcessor implements SpanProcessor {
```

Comment:
Reminder that this artifact is stable and we can't make breaking changes to any of the API surface area. Therefore, we should think carefully about this addition.

Comments:
- Make the class final
- Add a private constructor
- Add a `public static RemoteAttributesSpanProcessor create()` method for initializing.

Reply:
Updated.